test/builders/build/substrate-api/polkadotjs-api.js

```diff
@@ -56,22 +54,18 @@ describe('Polkadot.js API', function () {
 
   describe('Keyrings - Adding Accounts', async () => {
```

Comment:
Is there a docs PR to go along with this and update [these sections](https://docs.moonbeam.network/builders/substrate/libraries/polkadot-js-api/#creating-a-keyring-instance)? I'm not sure we should have docs on the Polkadot.js API page that show you how to create an account with Ethers, but I might be missing some additional context. Wdyt?

Reply:
Addressed here! https://github.com/moonbeam-foundation/moonbeam-docs/pull/1223